pandas/tests/copy_view/test_methods.py

```diff
@@ -1550,6 +1551,23 @@ def test_chained_where_mask(using_copy_on_write, func):
         with tm.raises_chained_assignment_error():
             getattr(df[["a"]], func)(df["a"] > 2, 5, inplace=True)
         tm.assert_frame_equal(df, df_orig)
+    else:
+        with tm.assert_produces_warning(
+            FutureWarning, match="inplace method", check_stacklevel=False
```

Comment:
Why is the check_stacklevel=False needed here? (the warnings are added to top-level methods, and we are testing those directly, so that should be a simple case?)

Reply:
I have no clue, this showed up in a ci build only and only on one, didn't want to spend more time debugging this...

But can try to reproduce locally if you want

Reply:
Ah, that's just because mask is calling where under the hood. So it was actually raising the warning twice (once with correct and once with incorrect stacklevel). 

Pushing a fix to let it only warn once.

Reply:
Ah interesting

Not clue why this didn't fail locally though